### PR TITLE
fix issues raised in #417

### PR DESCRIPTION
**Pull request by samhstn**
See #417

### CODE REVIEW

**Comment by iteles** [comment]
Linked to #386

**Review by nelsonic — APPROVED** [review]
@shouston3 👍

**Review by SimonLab — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 1]
```diff
@@ -157,23 +157,27 @@ If updating your environment variable doesn't work, try `rm -rf _build` and then
...
 
 ##### Hero background image
 
-The image should contrasts well with white text on top of it
+The image should contrasts well with white text on top of it.
+
+The image should be a wide, with a dimention ratio of approximately 3x2, try to aim for a pixel ratio between 500px and 1200px.
 
-The image should be a wide, with a dimention ratio of approximately 3x2, try to aim for a pixel ratio between 500px and 1200px
+Also note that on mobile the hero image is cut off so please test your image on the [staging area](https://ldmw-cms-staging.herokuapp.com/admin)
```
Comment (iteles) [inline README.md]: @shouston3 As discussed, let's use different terminology here to explain it is working as expected and not a bug - something along the lines of using a one size fits all image means it is displayed in different ways on each form factor as we have specified that it needs to cover the full background of the text it contains.

Let's also:
+ Add an example screenshot to the documentation 
+ Centre the image on mobile

**Review by iteles — CHANGES_REQUESTED** [review]
Thanks @shouston3, let's make the changes we discussed earlier today please 👍

File: README.md (hunk 1 above)
Comment (shouston3) [inline README.md]: I've just centered the image for every device, I think it looks better (much better on mobile though)

**Review by iteles — APPROVED** [review]
LGTM

**Pull request merged**